modules/aws/vault/variables.tf

```diff
@@ -53,6 +57,11 @@ variable "volume_size_etcd" {
   default = 10
 }
 
+variable "volume_size_logs" {
+  type    = "string"
+  default = 5
```

Comment:
Is it in GB?

What will happen when this volume becomes full?

Reply:
Yes, it is in gigabytes. If it is full - result will be the same as previously - vault will fail.

Reply:
So why separate partition? So we can SSH?

Do you want to make it bigger afterwards?

Reply:
Yes, we can adjust it later. But keeping logs separately is good idea in general.